dev-client/src/navigation/components/BottomNavButton.tsx

```diff
@@ -15,38 +15,40 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 
-import React from 'react';
-import {Pressable} from 'react-native';
+import {Pressable, PressableProps} from 'react-native';
 
 import {Center, IconButton as NativeIconButton} from 'native-base';
 
 import {Icon, IconName} from 'terraso-mobile-client/components/icons/Icon';
 import {Box, Text} from 'terraso-mobile-client/components/NativeBaseAdapters';
 
-export type IconButtonProps = React.ComponentProps<typeof NativeIconButton> & {
+export type BottomNavButtonProps = {
   name: IconName;
-  label?: string;
+  label: string;
+  onPress: PressableProps['onPress'];
 };
 
-export const IconButton = React.forwardRef(
-  ({name, label, ...props}: IconButtonProps, ref: React.Ref<unknown>) => {
-    const icon = (
-      <NativeIconButton ref={ref} icon={<Icon name={name} />} {...props} />
-    );
-    if (label === undefined) {
-      return icon;
-    }
-    return (
-      <Pressable onPress={props.onPress}>
-        <Box p="1">
-          {icon}
-          <Center>
-            <Text color="primary.contrast" fontSize="xs">
-              {label}
-            </Text>
-          </Center>
-        </Box>
-      </Pressable>
-    );
-  },
-);
+export const BottomNavButton = ({
```

Comment:
The old `BottomNavIconButton` was the only instance of the legacy `IconButton` that included label text. Discussing with design yielded the decision to separate the bottom nav controls from the `IconButton` designs, so I just inlined the old implementation and renamed the component to `BottomNavButton`.